app/uk/gov/hmrc/plasticpackagingtax/registration/views/start_page.scala.html

```diff
@@ -101,7 +101,7 @@ <h2 id="make-declaration" class="govuk-heading-m">@messages("startPage.makeDecla
 
     @govukButton(
         Button(
-            href = Some(RegistrationController.displayPage().url),
+            href = Some(pptRoutes.LiabilityWeightController.displayPage().url),
```

Comment:
This alias is not needed at all :sweat_smile: 
But on the IDE it will not present you any editor warnings. 
That is useful on the returns microservice where we distinguish between the account and returns sections.